Source/Orts.Simulation/Simulation/RollingStocks/TrainCar.cs

```diff
@@ -176,6 +180,79 @@ public abstract class TrainCar
         public float _AccelerationMpSS;
         protected IIRFilter AccelerationFilter = new IIRFilter(IIRFilter.FilterTypes.Butterworth, 1, 1.0f, 0.1f);
 
+        // Wheel Bearing Temperature parameters
+        public float WheelBearingTemperatureDegC = 40.0f;
+        public string DisplayWheelBearingTemperatureStatus;
+        public float WheelBearingTemperatureRiseTimeS = 0;
+        public float HotBoxTemperatureRiseTimeS = 0;
+        public float WheelBearingTemperatureDeclineTimeS = 0;
+        public float InitialWheelBearingDeclineTemperatureDegC;
+        public float InitialWheelBearingRiseTemperatureDegC;
+        public float InitialHotBoxRiseTemperatureDegS;
+        public bool WheelBearingFailed = false;
+        public bool WheelBearingHot = false;
+        public bool HotBoxActivated = false;
+        public bool HotBoxHasBeenInitialized = false;
+        public bool HotBoxSoundActivated = false;
+        public float HotBoxDelayS;
+        public float ActivityHotBoxDurationS;
+        public float ActivityElapsedDurationS;
+        public float HotBoxStartTimeS;
+
+        // Setup for ambient temperature dependency
+        Interpolator OutsideWinterTempbyLatitudeC;
+        Interpolator OutsideAutumnTempbyLatitudeC;
+        Interpolator OutsideSpringTempbyLatitudeC;
+        Interpolator OutsideSummerTempbyLatitudeC;
+        public bool AmbientTemperatureInitialised;
+
```

Comment:
Please describe the above variables adding a comment to each of them.

Reply:
Ok, done.